### ISSUE TITLE
Address security issues

### ISSUE THREAD

**Issue by enrichman** [epinio/epinio#2176]
Resolve reported security issues: https://github.com/epinio/epinio/security/dependabot

**Comment by andreas-kupries** [comment]
This resolves to bumping:

> require github.com/containerd/containerd v1.6.18
require github.com/docker/docker v20.10.24

These are indirect dependencies. Originators are:

|origin|requested|
|---|---|
|github.com/mittwald/go-helm-client@v0.11.5 |github.com/docker/docker@v20.10.18+incompatible|
|helm.sh/helm/v3@v3.11.1			   |github.com/docker/docker@v20.10.21+incompatible|
|oras.land/oras-go@v1.2.2		   |github.com/docker/docker@v20.10.21+incompatible|
|||
|github.com/mittwald/go-helm-client@v0.11.5 |github.com/containerd/containerd@v1.6.8|
|helm.sh/helm/v3@v3.11.1 		   |github.com/containerd/containerd@v1.6.15|
|oras.land/oras-go@v1.2.2 		   |github.com/containerd/containerd@v1.6.12|

Checking these for new releases.

**Comment by andreas-kupries** [comment]
Originators were found by inspecting the output of

```
go mod graph|grep docker/docker
go mod graph|grep containerd/containerd
```

**Comment by andreas-kupries** [comment]
Current `0.12.1` :: containerd bump since https://github.com/mittwald/go-helm-client/releases/tag/v0.11.6
Also bumps `helm.sh/helm/v3` (-> `3.11.2`). So maybe these too.

https://github.com/oras-project/oras-go/releases/tag/v1.2.3 - bumps docker/docker ...

helm.sh not clear

**Comment by andreas-kupries** [comment]
Ah, `oras-go` also indirect, through `helm.sh`. A mittwald bump might be enough.

**Comment by andreas-kupries** [comment]
The tickets needing a docker bump (`require github.com/docker/docker v20.10.24`) are not closed by #2187
